### ISSUE TITLE
Satpy resampling plot produce blank area which actually has values

### ISSUE THREAD

**Issue by J-shel** [pytroll/satpy#2144]
**Describe the bug**

Hi,
I resample the ABI L1b Radiance data into a different projection. After this, I plot it and generate a figure with blank area in the upper-left corner, which should not happen. And if I increase the projection area, the blank area gone away.

**To Reproduce**

#!/usr/bin/env python
import os
os.environ['OMP_NUM_THREADS'] = os.environ['MKL_NUM_THREADS'] = '2'
os.environ['PYTROLL_CHUNK_SIZE'] = '1024'

import warnings
warnings.simplefilter('ignore')

import dask
dask.config.set(scheduler='threads', num_workers=4)

from satpy import Scene
from glob import glob
from pyresample import create_area_def
import matplotlib.pyplot as plt

filenames = glob('OR_ABI-L1b-RadF-M3C01_G16_s20170590006095_e20170590016462_c20170590016506.nc')
scn = Scene(reader='abi_l1b', filenames=filenames)

my_channel = 'C01'
scn.load([my_channel])

my_area = create_area_def('my_area', {'proj': 'eqc', 'lon_0': -100, 'lat_0': 37.5, },
                          resolution=0.01,
                          area_extent=[-135, 25, -65, 50], units='degrees')           #Area 1: it will produce blank area in the upper-left corner 
###                    area_extent=[-140, 25, -65, 50], units='degrees')        # Area 2: No blank area is generated

new_scn = scn.resample(my_area)
print("------new scn------")
print(new_scn['C01'].shape)

plt.figure()
plt.imshow(new_scn['C01'])
plt.colorbar()
plt.show()

**Expected behavior**
I would like to know why, and hope no blank area is generated with the area "area_extent=[-135, 25, -65, 50], units='degrees')".

**Actual results**
No error messages generated.

**Screenshots**
I added the figure "Area_1_vs_blank_area" plotted with using the area_extent=[-135, 25, -65, 50], units='degrees'), which is Area 1, and the figure "Area_2" plotted with using the area_extent=[-140, 25, -65, 50], units='degrees'). Hope it helps.  

![Area_1_vs_blank_area](https://user-images.githubusercontent.com/82662111/178938596-db15ed0f-14cc-49ee-b497-3efc36a8b110.png)
![Area_2](https://user-images.githubusercontent.com/82662111/178938649-a7bbf1ed-6427-4aef-893f-4188a21990e5.png)

**Environment Info:**
 - OS: Linux
 - Satpy Version: 0.36.0
 - PyResample Version: 1.24.1
 - matplotlib Version: 3.5.2

**Comment by djhoese** [comment]
Yeah this is definitely odd. Could you try passing `reduce_data=False` to your `.resample` call and see if that fixes the issue?

**Comment by J-shel** [comment]
Hi,

Sorry for the late reply. "passing reduce_data=False to .resample call" works! Thank you very much. And besides, if not adding "reduce_data=False" and save the plot as png, the blank area also disappears, which is very interesting.

**Comment by TAlonglong** [comment]
Ths issuer seems to have solved the issue and no further updates has been received for more than 3 years. Closing.
